src/encryption.rs

```diff
@@ -116,6 +116,51 @@ impl From<matrix_sdk_crypto::types::EventEncryptionAlgorithm> for EncryptionAlgo
     }
 }
 
+#[wasm_bindgen()]
+#[derive(Debug, Clone, PartialEq, Eq)]
+/// Strategy to collect the devices that should receive room keys for the
+/// current discussion.
+///
+/// See matrix_sdk_crypto::CollectStrategy
```

Comment:
Please make these docs stand in their own right, so that the online doc (https://matrix-org.github.io/matrix-rust-sdk-crypto-wasm/classes/OlmMachine.html) is useful. (The fact that there are existing docs that just point to a Rust class is definitely bad.)

Reply:
done